src/tests/tests/yellowlist.js

```diff
@@ -0,0 +1,112 @@
+/* globals badger:false */
+
+(function() {
+
+  function get_ylist() {
```

Comment:
Filename too. Sorry for being pedantic, but I think the inconsistency can confuse people.